test/with_api_v2/acceptance.go

```diff
@@ -299,6 +332,42 @@ func (am *Alertmanager) Start() {
 	am.t.Fatalf("Starting alertmanager failed: timeout")
 }
 
+// WaitForCluster waits for the Alertmanager instance to join a cluster with the
+// given size.
+func (am *Alertmanager) WaitForCluster(size int) error {
+	params := general.NewGetStatusParams()
+	params.WithContext(context.Background())
+	var status general.GetStatusOK
+
+	// Poll for 2s
+	for i := 0; i < 20; i++ {
+		status, err := am.clientV2.General.GetStatus(params)
```

Comment:
👍 
another note: any plans on moving this into a top-level `client/v2` package?

Reply:
So that other projects would be able to vendor it? Great idea! I think this should take place in a follow up PR then though.